tests/testutils.py

```diff
@@ -213,6 +215,24 @@ def assert_run_python_script(source_code, timeout=TIMEOUT):
         os.unlink(source_file)
 
 
+def check_determinist_pickle(a, b):
+    """Check that two pickle output are the same.
+
+    If it is not the case, print the diff between the disaembled pickle.
+    This helper is useful to investigate non-determinist pickle.
+    """
+    if a != b:
+        with io.StringIO() as out:
+            pickletools.dis(pickletools.optimize(a), out)
+            a_out = out.getvalue()
+            a_out = "\n".join(ll[11:] for ll in a_out.splitlines())
```

Comment:
Could you please add a comment to explain why you discard the first 11 elements?